code/__DEFINES/subsystems.dm

```diff
@@ -125,6 +125,7 @@
 #define INIT_ORDER_NETWORKS 45
 #define INIT_ORDER_ECONOMY 40
 #define INIT_ORDER_OUTPUTS 35
+#define INIT_ORDER_RESTAURANT 34
```

Comment:
Does SSrestaurant depend on SSatoms? If there's no dependency, it should just have the default init order, and if it does have dependencies, they should be documented in the comments for the subsystem.

Reply:
ok